src/sentry/snuba/metrics/naming_layer/mri.py

```diff
@@ -0,0 +1,63 @@
+__all__ = ("SessionMRI", "TransactionMRI")
+
+from enum import Enum
+
+
+class SessionMRI(Enum):
```

Comment:
Could we include something here explaining what MRI stands for especially since we use it everywhere? Sorry if I missed it but I could only find MRI -> Metric Resource Identifier in notion

Reply:
Yeah that's a fair point.. will add doc.. fwiw that page in notion is the correct page